app/code/Magento/Vault/i18n/en_US.csv

```diff
@@ -13,3 +13,4 @@ Actions,Actions
 Delete,Delete
 "PayPal Account","PayPal Account"
 Cancel,Cancel
+"Are you sure you want to delete this card: %1?","Are you sure you want to delete this card: %1?"
```

Comment:
please add the new line at the end of the file, as it was before
```suggestion
"Are you sure you want to delete this card: %1?","Are you sure you want to delete this card: %1?"

```

Reply:
thanks, it's done!